INSTRUCTION.md

```diff
@@ -0,0 +1,10 @@
+### Apply the changes
+```
+kubectl apply -f confgiMap.yml
```

Comment:
There is a typo in the filename `confgiMap.yml`. It should be `configMap.yml` to match the correct file name.

Reply:
done